libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -360,18 +356,18 @@ void AC_AutoTune::control_attitude()
         // Initialize test-specific variables
         switch (axis) {
         case ROLL:
-            abort_angle = AUTOTUNE_TARGET_ANGLE_RLLPIT_CD;
+            abort_angle = attitude_control->lean_angle_max_cd() * AUTOTUNE_TARGET_ANGLE_RLLPIT;
             start_rate = ToDeg(ahrs_view->get_gyro().x) * 100.0f;
             start_angle = ahrs_view->roll_sensor;
             break;
         case PITCH:
-            abort_angle = AUTOTUNE_TARGET_ANGLE_RLLPIT_CD;
+            abort_angle = attitude_control->lean_angle_max_cd() * AUTOTUNE_TARGET_ANGLE_RLLPIT;
             start_rate = ToDeg(ahrs_view->get_gyro().y) * 100.0f;
             start_angle = ahrs_view->pitch_sensor;
             break;
         case YAW:
         case YAW_D:
-            abort_angle = AUTOTUNE_TARGET_ANGLE_YAW_CD;
+            abort_angle = attitude_control->lean_angle_max_cd() * AUTOTUNE_TARGET_ANGLE_YAW;
```

Comment:
This doesn't look right; your lean angle doesn't usually affect your yaw...

Reply:
This is a human factors thing. While lean angle doesn't affect yaw they are related in the expectations of the user. The pilot of a large aircraft with a small maximum lean angle is likely going to appreciate a smaller maximum yaw twitch angle too. This aircraft is also likely to need to reduce the twitch size anyway. So this change is likely to speed up the test and make the pilot feel more comfortable.

Reply:
I will add a comment.